Gemfile

```diff
@@ -5,6 +5,7 @@ gem 'sinatra'
 gem 'slim'
 
 group :development do
+  gem 'jekyll', '3.0.0'
```

Comment:
I thought we decided to use [pessimistic versioning](https://robots.thoughtbot.com/rubys-pessimistic-operator)?